src/components/auth/Login.js

```diff
@@ -0,0 +1,93 @@
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { FaUser, FaLock } from 'react-icons/fa';
+import { CgLogOut } from 'react-icons/cg';
+import { MdSupervisedUserCircle } from 'react-icons/md';
+import { Link, useNavigate } from 'react-router-dom';
+import { login } from '../../Redux/feature/UserSlice';
+import '../../styling/Auth.css';
+
+const Login = () => {
```

Comment:
- [ ] The registration and login are not consistent with the "Full name" and "UserName":

### Registration asks for a "Full name"

![image](https://github.com/codeWithHyder/house-rent-frontend/assets/57421823/ca2d8c85-7f6e-4220-9b83-7bf230032936)

### Login asks for a "UserName"

![image](https://github.com/codeWithHyder/house-rent-frontend/assets/57421823/0a6901f9-bf4e-426e-94ee-bd2ccbed50cf)

This is confusing because I just registered with my full name which could be 3-4 words, but now I am been asked for a "UserName" which normally is just 1 word.

To make it easier for your users, ask for their UserName in both the registration and login forms.

This is a matter of just changing the form labels, as the server is working fine.

That way, the registration and login forms will be more consistent and your UX will be improved.  💪